src/main/java/com/lpnu/shaggybeavers/mapper/RelicMapper.java

```diff
@@ -1,4 +1,53 @@
 package com.lpnu.shaggybeavers.mapper;
 
+import com.lpnu.shaggybeavers.dto.RelicDTO;
+import com.lpnu.shaggybeavers.model.Relic;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper
 public interface RelicMapper {
+
+    // Relic data
+//    @Mapping(source = "relic.name", target = "name")
+//    @Mapping(source = "relic.creationDate", target = "creationDate")
+    @Mapping(source = "relic.creationPlace.name", target = "creationPlaceName")
+//    @Mapping(source = "relic.picture", target = "picture")
+    @Mapping(source = "relic.museum.name", target = "museumName")
+    @Mapping(source = "relic.region.name", target = "regionName")
```

Comment:
Remove relic. prefix because if you use it like that mapstruct won't detect from what source it should map